src/javascript/lib/landing_pages/common.js

```diff
@@ -7,9 +7,11 @@ function checkWidth() {
 function toggleMobileMenu() {
     const toggleButton = document.getElementById('toggle-menu');
     const navbar       = document.getElementById('navigation');
+    const navbar_item  = document.getElementsByClassName('navbar-collapse')[0];
     toggleButton.addEventListener('click', function (e) {
-        navbar.classList[navbar.classList.contains('expand') ? 'remove' : 'add']('expand');
         e.stopPropagation();
+        navbar.classList[navbar.classList.contains('expand') ? 'remove' : 'add']('expand');
```

Comment:
@kellybinary  Since we are supprting `IE9+`, we can also use `navbar.classList.toggle('expand');` here.

Reply:
@aminroosta ok, i've changed to use `toggle`